apps/docs/docs/how-to-guides/faststore-api/using-graphiql-to-explore-the-faststore-api.mdx

```diff
@@ -5,19 +5,19 @@ sidebar_position: 1
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Using GraphiQL to explore the FastStore API
+# Using a GraphQL IDE to explore the FastStore API
```

Comment:
The rest of the documentation from "Building queries" explains how to use GraphiQL, so maybe it makes sense to keep the original title? Although it focuses on GraphiQL, with GraphQL Playground it should be pretty similar.